google/cloud/security/scanner/audit/org_rules_engine.py

```diff
@@ -269,22 +340,44 @@ def add_rule(self, rule_def, rule_index):
                                 bindings=rule_def.get('bindings', []),
                                 mode=rule_def.get('mode'))
 
+                    applies_to = resource.get('applies_to')
+
                     # See if we have a mapping of the resource and rule
-                    resource_rules = self.resource_rules_map.get(gcp_resource)
+                    resource_rules = self.resource_rules_map.get(
+                        (gcp_resource, applies_to))
 
                     # If no mapping exists, create it.
                     if not resource_rules:
                         resource_rules = ResourceRules(
                             resource=gcp_resource,
-                            applies_to=resource.get('applies_to'),
+                            applies_to=applies_to,
```

Comment:
This applies_to=applies_to is not very clear to read/understand. Can you look at improving either the naming or code block to make this more readable?